### DISCUSSION TITLE
Operating environment

### DISCUSSION THREAD

**Discussion by ZisongXu** [NVlabs/curobo#37] [Software Q&A] [answered]
I would like to ask if it is possible to run CuRobo on Windows without isaac_sim?

**Comment by balakumar-s** [comment]
If you can run pytorch in windows, cuRobo should work as it mainly relies on pytorch and CUDA to compile the kernels. However, this has not been tested.

**Comment by Spphire** [comment, marked as answer]
it can be used by wsl2 in win11.
but if u want to use custom robot, u need to configure it in isaac-sim in linux

**Comment by anubhav-dogra** [comment]
Is there any simplistic example for this ? 
I was trying to install through docker in windows, but failed at command number 9, I guess

**Reply by myrtenasty** [reply]
You can run the `build_docker.sh` script in `curobo/docker` from wsl2 on windows if you have docker desktop installed. This should be the easiest way to try curobo on windows
